apps/judicial-system/api/src/app/modules/backend/backend.service.ts

```diff
@@ -298,7 +298,15 @@ export class BackendService extends DataSource<{ req: Request }> {
     query: SubpoenaStatisticsInput,
   ): Promise<SubpoenaStatistics> {
     const searchParams = this.serializeNestedObject(query)
-    return this.get(`cases/subpoenas/statistics?${searchParams.toString()}`)
+    return this.get(`cases/subpoenas/statistics?${searchParams}`)
+  }
+
+  // TODO: Split
+  getPreprocessedDataCsvSignedUrl(
+    query: RequestStatisticsInput,
+  ): Promise<SignedUrl> {
+    const searchParams = this.serializeNestedObject(query)
+    return this.get(`cases/requests/statistics/export-csv?${searchParams}`)
```

Comment:
We might just end up using one endpoint for exports and let the controller control which service methods to call